investor_wallet_platform_base/__manifest__.py

```diff
@@ -41,6 +41,7 @@
         'report/cooperator_certificat.xml',
         'report/cooperator_to_certificat.xml',
         'report/easy_my_coop_report.xml',
+        'data/cron.xml',
         'report/cooperator_invoice.xml',
         'data/mail_template_data.xml'
```

Comment:
could you group files by nature whenever its possible ?

Reply:
Trop d'accord avec toi mais ça génère un conlift dans gitaggregator :-(